vnext/Microsoft.ReactNative.Cxx/JSValueReader-WinRT.h

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <CppWinRTIncludes.h>
+#include <UI.Xaml.Markup.h>
+#include <chrono>
+
+#include "JSValue.h"
+#include "JSValueTreeReader.h"
+#include "StructInfo.h"
+
+#include "winrt/Microsoft.ReactNative.h"
+
+#include <string>
+
+namespace winrt::Microsoft::ReactNative {
+inline BYTE GetAFromArgb(DWORD v) {
```

Comment:
>GetAFromArgb [](start = 12, length = 12)

It would be nice to have all function prototypes listed on the top of the file along with the short description using //! comment tags. This way we can auto-generate docs for them, use it in the IntelliSense and understand what these functions do, while the implementation below could be free from any comment-related clutter.  I.e. the same style as  in the JSValueReader.h